ntbs-service/Jobs/DataQualityAlertsJob.cs

```diff
@@ -125,7 +125,7 @@ public async Task Run(IJobCancellationToken token)
                 {
                     NotificationId = notification.NotificationId,
                     DuplicateId = notification.DuplicateId,
-                    NhsNumberMatch = notification.NhsNumberMatch
+                    NhsNumberMismatch = notification.NhsNumber != notification.DuplicateNhsNumber && notification.NhsNumber != null
```

Comment:
I think we still need `notification.DuplicateNhsNumber != null` as well here?

Reply:
If it gets to the secondcondition `notification.NhsNumber != null` we know the numbers match from the first condition. The second condition is only looked at in the case they're both null or neither are null and they match. The second condition just makes sure in the null case we don't say they are matching, as we know they're equal we can just check one of them for null.